modules/gitops-logging-into-keycloak.adoc

```diff
@@ -0,0 +1,53 @@
+[id="logging-into-keycloak_{context}"]
+= Logging into Keycloak
+
+.Procedure
+
```

Comment:
Possible introduction: 

`Logging into Keycloak gives you access to <state benefits or features of Keycloak>.`

Reply:
If you rewrite the introduction as I indicate above, I don't think an introduction here would be necessary.